docs/zdgbook/ObjectPublishing.rst

```diff
@@ -348,10 +329,10 @@ base with a *base* tag in your ``index_html`` method output.
 Response Headers
 ----------------
 
-The publisher and the web server take care of setting response headers
-such as *Content-Length* and *Content-Type*. Later in the chapter
-you'll find out how to control these headers and also how exceptions
-are used to set the HTTP response code.
+The publisher and the web server take care of setting the *Content-Length*
```

Comment:
You have removed `Content-Type` from the description; but `Content-Type` is still set just in a different way. I suggest to readd `Content-Type` to the description.